src/test/java/com/google/jenkins/plugins/computeengine/integration/ComputeEngineCloudRestartPreemptedIT.java

```diff
@@ -57,17 +57,15 @@
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
 import org.junit.Test;
-import org.junit.rules.Timeout;
 import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.recipes.WithTimeout;
 
 /**
  * Integration test suite for {@link ComputeEngineCloud}. Verifies that instances with preempted
  * flag will be restarted when preempted.
  */
 @Log
 public class ComputeEngineCloudRestartPreemptedIT {
```

Comment:
Actually there is more integration tests that are failing because of reasons like,

* they are depending on the debian-9 image (in the casc bundle the image path is there like that)
* and they are depending the JDK8 whose deb repository is not existing and giving 404 errors
etc like that..

So I will create a separate PR and fix all the integration tests. I will remove the existing integration test fixing code from here.

Once I file the other PR, I will link back in this comment's follow-up

Reply:
Integration test update PR is created here
https://github.com/jenkinsci/google-compute-engine-plugin/pull/495